napari/keybindings.py

```diff
@@ -138,3 +138,13 @@ def toggle_ndisplay(viewer):
         viewer.grid_view()
     else:
         viewer.stack_view()
+
+
+@Viewer.bind_key('Control-Alt-P')
+def play(viewer):
+    """Toggle animation on the first axis"""
+    if viewer.window.qt_viewer.dims.is_playing:
+        viewer.window.qt_viewer.dims.stop()
+    else:
+        # TODO: need to allow control over FPS in gui
+        viewer.window.qt_viewer.dims.play_dim(0, 10)
```

Comment:
can this use the `last_used` axis instead of being hardcoded to `0` - 
```python
axis = viewer.window.qt_viewer.dims.last_used
if axis is not None:
    viewer.window.qt_viewer.dims.play_dim(axis, 10)
```